docs/tutorials/TracingCode.md

```diff
@@ -150,12 +155,12 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
        ...
    ```
 
-1. _Step over_ the statements in that method until you reach the `switch` statement. The 'Variables' window now shows the value of both `commandWord` and `arguments`:<br>
+10. _Step over_ the statements in that method until you reach the `switch` statement. The 'Variables' window now shows the value of both `commandWord` and `arguments`:<br>
     ![Variables](../images/tracing/Variables.png)
 
-1. We see that the value of `commandWord` is now `edit` but `arguments` is still not processed in any meaningful way.
+11. We see that the value of `commandWord` is now `edit` but `arguments` is still not processed in any meaningful way.
 
-1. Stepping through the `switch` block, we end up at a call to `EditCommandParser().parse()` as expected (because the command we typed is an edit command).
+12. Stepping through the `switch` block, we end up at a call to `EditCommandParser().parse()` as expected (because the command we typed is an edit command).
```

Comment:
```suggestion
1. Stepping through the `switch` block, we end up at a call to `EditCommandParser().parse()` as expected (because the command we typed is an edit command).
```